casper/src/test/scala/coop/rchain/casper/batch1/MultiParentCasperRholangSpec.scala

```diff
@@ -66,18 +70,22 @@ class MultiParentCasperRholangSpec extends AnyFlatSpec with Matchers with Inspec
            |}
          """.stripMargin
 
-      def calculateUnforgeableName(timeStamp: Long): String =
-        Base16.encode(
-          Tools.unforgeableNameRng(Secp256k1.toPublic(ConstructDeploy.defaultSec), timeStamp).next()
-        )
+      def calculateUnforgeableName(block: BlockMessage): String = {
```

Comment:
This calculation is for deploy so maybe rename it to `calculateDeployUnforgeableName`.